charts/sonarqube-lts/README.md

```diff
@@ -1,6 +1,8 @@
-# SonarQube
+# SonarQube LTS (Deprecated)
 
-Long-term Support for stability. Improve Code Quality and Code Security throughout your workflow for up to 27 languages. [SonarQube](https://www.sonarqube.org/) can detect Bugs, Vulnerabilities, Security Hotspots and Code Smells and give you the guidance to fix them.
+> **DEPRECATION NOTICE**: Since the latest SonarQube LTS (9.9.0), this chart has been deprecated, is no longer maintained and is replaced by the standard [sonarqube chart](https://artifacthub.io/packages/helm/sonarqube/sonarqube). **Please read the upgrade notes available at [this page](https://artifacthub.io/packages/helm/sonarqube/sonarqube#upgrade-from-sonarqube-lts-to-this-chart) to install the current LTS chart.**
+
+Improve Code Quality and Code Security throughout your workflow for up to 27 languages. [SonarQube](https://www.sonarqube.org/) can detect Bugs, Vulnerabilities, Security Hotspots and Code Smells and give you the guidance to fix them.
 
 ## Introduction
```

Comment:
I recommend updating the sentence below with:

> This chart bootstraps a SonarQube 8.9 (previous LTS) instance with a PostgreSQL database. If you want to use the latest version of SonarQube or the new SonarQube 9.9 LTS, please use the [sonarqube](../sonarqube/) helm chart. Deciding between LTS and Latest? [This may help](https://www.sonarsource.com/products/sonarqube/downloads/lts/)
```